src/components/Map/Map.js

```diff
@@ -114,6 +124,13 @@ class Map extends PureComponent {
     dispatchHtmlEvent(htmlEvent);
   }
 
+  onPointerDown() {
+    const { map } = this.props;
+    if (document.activeElement !== map.getTarget()) {
+      map.getTarget().focus();
```

Comment:
To be sure you ve got the map element and not a string

```suggestion
      map.getTargetElement().focus();
```